armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -345,49 +346,99 @@ def getLabel(caseTitle, cycle, node, iteration=None):
 
 
 class GlobalFluxOptions(executers.ExecutionOptions):
-    """Data structure representing common options in Global Flux Solvers"""
+    """Data structure representing common options in Global Flux Solvers
+
+    Attributes
+    ----------
+    adjoint : bool
+        True if the ``CONF_NEUTRONICS_TYPE`` setting is set to ``adjoint``.
+    calcReactionRatesOnMeshConversion : bool
+        This option is used to recalculate reaction rates after a mesh
+        conversion and remapping of neutron flux. This can be disabled
+        in certain global flux implementations if reaction rates are not
+        required, but by default it is enabled.
+    eigenvalueProblem : bool
+        Whether this is a eigenvalue problem or a fixed source problem
+    includeFixedSource : bool
+        This can happen in eig if Fredholm Alternative satisfied.
+    photons : bool
+        Run the photon/gamma uniform mesh converter?
+    real : bool
+        True if  ``CONF_NEUTRONICS_TYPE`` setting is set to ``real``.
+    aclpDoseLimit : float
+        Dose limit in dpa used to position the above-core load pad (if one exists)
+    boundaries : str
+        External Neutronic Boundary Conditions. Reflective does not include axial.
+    cs : Settings
+        Settings for this run
+    detailedAxialExpansion : bool
+        Turn on detailed axial expansion? from settings
+    dpaPerFluence : float
+        A quick and dirty conversion that is used to get dpaPeak by multiplying
+        the factor and fastFluencePeak
+    energyDepoCalcMethodStep : str
+        For gamma transport/normalization
+    epsEigenvalue : float
+        Convergence criteria for calculating the eigenvalue in the global flux solver
+    epsFissionSourceAvg : float
+        Convergence criteria for average fission source, from settings
+    epsFissionSourcePoint : float
+        Convergence criteria for point fission source, from settings
+    geomType : geometry.GeomType
+        Reactor Core geometry type (HEX, RZ, RZT, etc)
+    hasNonUniformAssems: bool
+        Has any non-uniform assembly flags, from settings
+    isRestart : bool
+        Restart global flux case using outputs from last time as a guess
+    kernelName : str
+        The neutronics / depletion solver for global flux solve.
+    loadPadElevation : float
+        The elevation of the bottom of the above-core load pad (ACLP) from
+        the bottom of the upper grid plate (in cm).
+    loadPadLength : float
+        The length of the load pad. Used to compute average and peak dose.
+    maxOuters : int
+        XY and Axial partial current sweep max outer iterations.
+    savePhysicsFilesList : bool
```

Comment:
`savePhyiscsFilesList` is the list of timestamps from the settings file. `self.savePhysicsFiles` is a boolean derived from `self.savePhysicsFilesList`

Reply:
(This is the "Attributes" section of the docstring, so everything listed here is a "self.something".)